src/AddNode.scss

```diff
@@ -47,30 +47,6 @@ button.btn.AddNode__expanded-button svg {
     stroke: #FFF;
 }
 
-.AddNode__drop-area-container {
-    position: relative;
-}
-
-.AddNode--is-dragging-command .AddNode__expanded-drop-area {
-    top: calc(
-        #{-$addNodeDropAreaHeight / 2}
-        + #{$heightOfExpandedIcon / 2}
-        + #{$addNodeExpandedButtonBorderWidth}
-        + #{$addNodeExpandedButtonPadding});
-    left: -$addNodeDropMargin;
-    position: absolute;
-    width: calc(
-        #{2 * $addNodeDropMargin}
-        + #{$widthOfExpandedIcon}
-        + #{2 * $addNodeExpandedButtonBorderWidth}
-        + #{2 * $addNodeExpandedButtonPadding});
-    height: $addNodeDropAreaHeight;
-}
-
-.AddNode--is-dragging-command .AddNode__collapsed-drop-area {
-    top: (-$addNodeDropAreaHeight / 2) + ($heightOfCollapsedIcon / 2);
-    left: -$addNodeDropMargin;
-    position: absolute;
-    width: (2 * $addNodeDropMargin) + $widthOfCollapsedIcon;
-    height: $addNodeDropAreaHeight;
+.AddNode__expanded-button--isDragTarget {
+    box-shadow: 0 0 0 0.4rem #53A28C !important;
```

Comment:
Rather than using `!important`, please make the selector more specific:

    .AddNode__expanded-button.AddNode__expanded-button--isDragTarget {
        box-shadow: 0 0 0 0.4rem #53A28C;
    }

Using `!important` will make applying themes more difficult as we will have to deal with relative importance in addition to relative specificity of selectors.

Reply:
Fixed.